assemblyai.go

```diff
@@ -6,13 +6,14 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"mime"
 	"net/http"
 	"net/url"
 	"os"
 )
 
 const (
-	version              = "1.8.0"
```

Comment:
@he-james the version here seems to be higher than in our monorepo.

Reply:
I agree, thinking I should update to 1.8.1?